x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/privilege_summary/privilege_summary_table.test.tsx

```diff
@@ -40,15 +40,19 @@ const spaces = [
   },
 ];
 
-const expectNoPrivileges = (displayedPrivileges: any) => {
+const maybeExpectSubFeaturePrivileges = (expect: boolean, subFeaturesPrivileges: unknown) => {
```

Comment:
The diff doesn't show it, but most of these tests are the same, with the distinction that they've been updated to run with both sub-features enabled and disabled.